WalkingChallenge/SettingsViewController.swift

```diff
@@ -0,0 +1,205 @@
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ **/
+
+import Foundation
+import FacebookLogin
+import UIKit
+import SnapKit
+
+fileprivate class PedometerCell: UITableViewCell, IdentifiedUITableViewCell {
+  static let identifier: String = "PedometerCell"
+
+  internal var devicePicture: UIImageView = UIImageView()
+  internal var deviceName: UILabel = UILabel(.header)
+  internal var deviceDescription: UILabel = UILabel(.body)
+
+  override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    initialise()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  private func initialise() {
+    addSubviews([devicePicture, deviceName, deviceDescription])
+
+    devicePicture.snp.makeConstraints { (make) in
+      make.top.equalToSuperview().inset(Style.Padding.p12)
+      make.left.equalToSuperview().inset(Style.Padding.p12)
+      make.size.equalTo(Style.Size.s56)
+      make.bottom.equalToSuperview().inset(Style.Padding.p12)
+    }
+
+    deviceName.snp.makeConstraints { (make) in
+      make.top.equalToSuperview().inset(Style.Padding.p12)
+      make.left.equalTo(devicePicture.snp.right).offset(Style.Padding.p8)
+    }
+
+    deviceDescription.textColor = Style.Colors.grey
+    deviceDescription.snp.makeConstraints { (make) in
+      make.top.equalTo(deviceName.snp.bottom)
+      make.left.equalTo(deviceName.snp.left)
+    }
+  }
+}
+
+private enum TableSection: Int {
+  case device = 0
+  case accountSettings
+  case teams
+  case helpAndSupport
+}
+
+struct SettingsInfo {
+  let title: String
+  let values: [SettingsnValue]
+}
+
+struct SettingsnValue {
+  let configValue: String
+}
+
+class SettingsSectionDataSource {
+
+  let pedometerSource = UserInfo.pedometerSource
+
+  // swiftlint:disable line_length
+  var settingsInfo: [SettingsInfo] = [
+    SettingsInfo(title: Strings.Settings.accountSettings,
+                  values: [SettingsnValue.init(configValue: Strings.Settings.editProfile),SettingsnValue.init(configValue: Strings.Settings.changeEmailAddress),SettingsnValue.init(configValue: Strings.Settings.notificationsAndReminders)]),
+    SettingsInfo(title: Strings.Settings.teams, values: [SettingsnValue.init(configValue: Strings.Settings.changeTeams)]),
+    SettingsInfo(title: Strings.Settings.helpAndSupport, values: [SettingsnValue.init(configValue: Strings.Settings.faq),SettingsnValue.init(configValue: Strings.Settings.contactUs)])
+    ]
+  // swiftlint:enable Line Length
+}
+
+fileprivate class SettingsDataSource: NSObject, UITableViewDataSource, UITableViewDelegate {
+
+  internal var dataSource: SettingsSectionDataSource?
+
+  override init() {
+    super.init()
+    self.dataSource = SettingsSectionDataSource()
+  }
+
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    if section == TableSection.device.rawValue {
+      return 1
+    } else {
```

Comment:
I mean to kill just the `else` keyword.

```
if section == TableSection.device.rawValue {
  return 1
}
let configuration = dataSource?.settingsInfo[section - 1]
return (configuration?.values.count)!
```

I think you can also remove the forced unwrap by doing this:

    return dataSource?.settingsInfo[safe: section - 1]?.values.count ?? 1